src/main/java/com/example/main/CashDrawer.java

```diff
@@ -97,12 +120,49 @@ public void takeBillsFromRegister(int twenty, int ten, int five, int two, int on
         this.one -= one;
     }
 
-    private boolean getAmountOfEachTotal(int changeAmount){
-        return false;
-    }
-
-    public void makeChange(int changeAmount){
-        getAmountOfEachTotal(changeAmount);
+    public boolean validateReturn(int changeLeftOver){
+        if(changeLeftOver > 0){
+            return false;
+        } return true;
+    }
+
+    public int[] makeChange(int changeAmount) throws InvalidAmountException {
```

Comment:
I can think of a few situations where this brute force change method won't work. Can you think of any?